data/rulesets.js

```diff
@@ -854,6 +855,12 @@ let BattleFormats = {
 			}
 		},
 	},
+	mimicglitch: {
+		effectType: 'ValidatorRule',
+		name: 'Mimic Glitch',
+		desc: "Allows any Pokemon with access to Assist, Copycat, Metronome, Mimic, or Transform to gain access to almost any other move.",
+		// Implemented in sim/team-validator.ts
```

Comment:
I assume this was already implemented, it's just that the rulesets entry was deleted at some point?

Reply:
Yes